conf/config.properties

```diff
@@ -57,7 +57,7 @@ buildServer=http://zre-matrix.eng.zimbra.com
 driveServer=http://zqa-257.eng.zimbra.com
 
 # Browser drivers
-chromeDriverURL=http://chromedriver.storage.googleapis.com/91.0.4472.19
+chromeDriverURL=http://chromedriver.storage.googleapis.com/93.0.4577.82
 geckoDriverURL=https://github.com/mozilla/geckodriver/releases/download/v0.26.0
```

Comment:
Please update gecko version as well for firefox.

Reply:
Updated geckoDriverURL as well.